Content.Server/Mobs/Roles/Job.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using Content.Server.Interfaces.Chat;
+using Content.Shared.Jobs;
+using Robust.Shared.IoC;
+
+namespace Content.Server.Mobs.Roles
+{
+    public class Job : Role
+    {
+        private readonly JobPrototype _jobPrototype;
+
+        public Job(Mind mind, JobPrototype jobPrototype) : base(mind)
+        {
+            _jobPrototype = jobPrototype;
+            Name = jobPrototype.Name;
+        }
+
+        public override string Name { get; }
```

Comment:
Put the variables at the top of the class to make it easier to read. Try to organize them like the class it inherits from.